notebooks/run_skywalker.ipynb

```diff
@@ -112,37 +119,53 @@
     "    s = source.Undulator('test_undulator')\n",
     "    m1 = mirror.OffsetMirror('test_m1h', z=90.510, alpha=0.0014)\n",
     "    m2 = mirror.OffsetMirror('test_m2h', x=0.0317324, z=101.843, alpha=0.0014)\n",
+    "    xrtm2 = mirror.OffsetMirror('test_xrtm2', x=0.0317324, z=200, alpha=0.0014)\n",
     "    y1 = pim.PIM('test_p3h', x=0.0317324, z=103.660,\n",
     "             zero_outside_yag= not infinite_yag)\n",
     "    y2 = pim.PIM('test_dg3', x=0.0317324, z=375.000,\n",
     "             zero_outside_yag= not infinite_yag)\n",
+    "    mecy1 = pim.PIM('test_mecy1', x=0.0317324, z=350,\n",
+    "             zero_outside_yag= not infinite_yag)\n",
+    "    mfxdg1 = mecy1\n",
     "    patch_pims([y1, y2], mirrors=[m1, m2], source=s)\n",
+    "    patch_pims([mecy1], mirrors=[xrtm2], source=s)\n",
     "\n",
     "    #Add noise\n",
     "    y1.centroid_noise = centroid_noise\n",
-    "    y2.centroid_noise = centroid_noise\n"
+    "    y2.centroid_noise = centroid_noise\n",
+    "    mecy1.centroid_noise = centroid_noise\n",
+    "\n",
+    "alignment_yags = dict(HOMS=[y1, y2],\n",
+    "                      MEC=[mecy1],\n",
+    "                      MFX=[mfxdg1],\n",
+    "                      XCS=['pbty', 'xcssb1y'])\n",
+    "alignment_mots = dict(HOMS=[m1, m2],\n",
+    "                      MEC=[xrtm2],\n",
+    "                      MFX=[xrtm2],\n",
+    "                      XCS=['xrtm1', 'xrtm3'])"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 4,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "#Create Skywalker plan\n",
-    "plan = skywalker([y1, y2], [m1, m2], 'detector_stats2_centroid_x', 'pitch',\n",
-    "             goals, first_steps=first_steps, tolerances=tolerances,\n",
-    "             averages=average, timeout=10)\n"
```

Comment:
This would require time that we don't have at the moment for commissioning of the cameras and changes on the pcds-devices... 
They will fail at stage() if we don't add the plugins to the class.